shared/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/hapi/HapiHelperTest.groovy

```diff
@@ -601,6 +601,39 @@ class HapiHelperTest extends Specification {
         HapiHelper.getORC21Value(sr) == null
     }
 
+
+    // OBR-2 - Placer Order Number
+    def "obr-2 methods work as expected"() {
```

Comment:
I think this test could use a better name

Reply:
This test is covering all the OBR-2 related methods in HapiHelper. Other tests in this file follow similar pattern, so if we change this one we should change the other ones too. Do you have any suggestions?

Reply:
Maybe obr-2 sets value to input or something?

Reply:
It's also testing the getters. What about `obr-2 getter and setters methods work as expected`? The `work as expected` is also a pattern we've been using for naming tests

Reply:
Sure, or you can leave it if you think it's fine